### PR TITLE
Fix/mms

### PR DESCRIPTION
**Pull request by villenavg**

### CODE REVIEW

File: OWDTestToolkit/apps/dialer.py [hunk 1]
```diff
@@ -317,13 +317,12 @@ def createMultipleCallLogEntries(self, phone_number, entries_number):
...
         today = datetime.datetime.today()
         for i in range(entries_number):
-            delta = datetime.timedelta(days=i)
-            new_date = today - delta
-
-            self.UTILS.date_and_time.setTimeToSpecific(p_day=new_date.day, p_month=new_date.month)
-
             self.enterNumber(phone_number)
             self.call_this_number_and_hangup(delay=5)
+            # This needs to be done bcs sometimes (50%) the Dialer app crushes after hanging up
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/dialer.py]: OMG!

File: OWDTestToolkit/apps/messages.py [hunk 2]
```diff
@@ -285,7 +285,7 @@ def create_and_send_mms(self, attached_type, nums, m_text):
             #
             # Load an video file into the device.
             #
-            self.UTILS.general.add_file_to_device('./tests/_resources/mpeg4.mp4', destination='/SD/mus')
+            self.UTILS.general.add_file_to_device('./tests/_resources/mpeg4.mp4', destination='DCIM/100MZLLA')
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/messages.py]: uploading an mp4 into the camera photos directory?

File: OWDTestToolkit/apps/dialer.py (hunk 1 above)
Comment (villenavg) [inline OWDTestToolkit/apps/dialer.py]: :dancer:

File: OWDTestToolkit/pixi_testcase.py [hunk 3]
```diff
@@ -8,4 +8,3 @@ def setUp(self):
         self.data_layer.set_setting('ums.enabled', False)
         self.data_layer.set_setting('ums.mode', 0)
         self.data_layer.set_setting('ums.status', 0)
-        self.data_layer.set_setting('keyboard.current', 'es')
```
Comment (FranciscoCano) [inline OWDTestToolkit/pixi_testcase.py]: Doh!

**Comment by FranciscoCano** [comment]
LGTM

**Pull request merged**